lib/stripe/customer.rb

```diff
@@ -10,6 +10,8 @@ class Customer < APIResource
 
     OBJECT_NAME = "customer".freeze
 
+    custom_method :delete_discount, http_verb: :delete, http_path: "discount"
+
```

Comment:
So I get why it's returning a `Discount`. But the fact that this one behaves differently from the instance one is quite tricky right? I can't think of a way to fix this but wanted to say so.

Reply:
Yeah, the inconsistency is mildly annoying, but I think it's for the best. If users want to retrieve the updated customer, they should do so explicitly. We should call this out in the changelog or migration guide though.